pages/claimstatus/index.tsx

```diff
@@ -1,15 +1,32 @@
+import fs from 'fs'
+import path from 'path'
+import https from 'https'
+import { promisify } from 'util'
+
 import Head from 'next/head'
 import Container from 'react-bootstrap/Container'
 import { ReactElement } from 'react'
 import { useTranslation } from 'next-i18next'
 import { serverSideTranslations } from 'next-i18next/serverSideTranslations'
 import { GetServerSideProps } from 'next'
+import pem from 'pem'
+import fetch from 'node-fetch'
 
 import { Header } from '../../components/Header'
 import { Main } from '../../components/Main'
 import { Footer } from '../../components/Footer'
 
-export default function Home(): ReactElement {
+
+export interface Claim {
+  ClaimType: (string | "not working")
```

Comment:
ClaimType is referenced here and in another diff line, I can't tell why since it's not used. Can remove?

Reply:
ClaimType is a field in the API responses we get back that are passed to the React component.  This is TypeScript at work, basically.